NEWS.md

```diff
@@ -132,6 +132,7 @@
   with `ByRow` is now accepted ([#2476](https://github.com/JuliaData/DataFrames.jl/pull/2476))
 * add `permutedims` method for `AbstractDataFrame` ([#2447](https://github.com/JuliaData/DataFrames.jl/pull/2447))
 * add support for `Cols` from DataAPI.jl ([#2495](https://github.com/JuliaData/DataFrames.jl/pull/2495))
+* added support for casting `AbstractDataFrame` to `Dict` datatype. [#2624](https://github.com/JuliaData/DataFrames.jl/pull/2624)
```

Comment:
Please format this in this way and add it to 1.0 release notes (now it is in 0.22 release notes; thank you)
```suggestion
* added support for casting `AbstractDataFrame` to `Dict` datatype
  ([#2624](https://github.com/JuliaData/DataFrames.jl/pull/2624))
```